### PR TITLE
feat(TV Time): recode & add buttons, covers & privacy

### PR DESCRIPTION
**Pull request by DarkVillager**
## Description 

Recoded the presence & added buttons, covers, privacy with their relevant settings.

## Acknowledgements
- [X] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [X] I linted the code by running `yarn format`
- [X] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

File: websites/T/TV Time/presence.ts [hunk 1]
```diff
@@ -3,73 +3,191 @@ const presence = new Presence({
 	}),
 	browsingTimestamp = Math.floor(Date.now() / 1000);
 
+enum Assets {
+	logo = "https://i.imgur.com/KuPkyx2.png",
+	searchImage = "https://i.imgur.com/oGQtnIY.png",
```
Comment (EncryptedDev) [inline websites/T/TV Time/presence.ts:8]:
```suggestion
	Logo = "https://i.imgur.com/KuPkyx2.png",
	SearchImage = "https://i.imgur.com/oGQtnIY.png",
```

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/T/TV Time/presence.ts (hunk 1 above)
Comment (EncryptedDev) [reply websites/T/TV Time/presence.ts]: Update references of the enum to resolve CI

File: websites/T/TV Time/presence.ts [hunk 2]
```diff
@@ -3,73 +3,191 @@ const presence = new Presence({
...
+	else {
+		switch (splitPathname[2]) {
+			case "show": {
+				switch (splitPathname[3]) {
+					case "explore": {
+						if (!document.querySelector('[id="shows-results"]')) {
+							presenceData.details = "Exploring Shows";
+							presenceData.buttons = [
+								{
+									label: "Exlpore Shows",
```
Comment (DarkVillager) [inline websites/T/TV Time/presence.ts:48]:
```suggestion
									label: "Explore Shows",
```

File: websites/T/TV Time/presence.ts [hunk 3]
```diff
@@ -3,73 +3,191 @@ const presence = new Presence({
...
+							document.querySelector('[class="day-number"]').textContent
+						} (${document
+							.querySelector('[class="relative-date"]')
+							.textContent.trim()})`;
+						presenceData.buttons = [
+							{
+								label: "View Their Calendar",
+								url: href,
+							},
+						];
```
Comment (EncryptedDev) [inline websites/T/TV Time/presence.ts:175]: Can you give me an example of viewing someone else's calendar? Seems too privacy invasive

Comment (DarkVillager) [reply websites/T/TV Time/presence.ts:175]: > Can you give me an example of viewing someone else's calendar? Seems too privacy invasive

image

Comment (EncryptedDev) [reply websites/T/TV Time/presence.ts:175]:
```suggestion
						presenceData.buttons = [
							{
								label: "View Calendar",
								url: href,
							},
						];
```

**Review by EncryptedDev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**